common/display/displayplane.cpp

```diff
@@ -126,13 +127,13 @@ bool DisplayPlane::Initialize(uint32_t gpu_fd,
   if (!ret)
     ETRACE("Could not get alpha property");
 
-#ifndef DISABLE_EXPLICIT_SYNC
-  ret = in_fence_fd_prop_.Initialize(gpu_fd, "IN_FENCE_FD", plane_props);
-  if (!ret)
-    ETRACE("Could not get IN_FENCE_FD property");
-#else
-  in_fence_fd_prop_.id = 0;
-#endif
+  if (android::DrmHwcTwo::IsExplicitSyncEnabled()) {
```

Comment:
white space ? common/ should not include any android specific or linux specific headers